src/tribol/interface/tribol.hpp

```diff
@@ -144,6 +144,13 @@ void setPlotOptions( enum VisType v_type );
  */
 void setOutputDirectory( const std::string& dir );
 
+/*!
+ * \brief Sets the logging level
+ * \param [in] csId coupling scheme id
+ * \param [in] log_level the desired logging level 
+ */
+void setLoggingLevel( const int csId, const LoggingLevel log_level );
```

Comment:
Since Tribol has a hard dependency on axom and slic, any reason for it to define it's own `LoggingLevel` enum rather than use the slic one? If this planning for future extensions?

Reply:
I was staying flexible by doing this, but I don't see any need for a future extension. Modify to use the slic namespace logging levels?

Reply:
I think that makes sense -- since this PR is time-sensitive perhaps we can make that change in a follow-up PR.

Reply:
We will keep the new enum for now per design discussion.